feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/data/network/blockhain/model/DelayedThresholdPassing.kt

```diff
@@ -0,0 +1,21 @@
+package io.novafoundation.nova.feature_governance_api.data.network.blockhain.model
+
+import io.novafoundation.nova.common.data.network.runtime.binding.Perbill
+import io.novafoundation.nova.feature_governance_api.data.network.blockhain.model.DelayedThresholdPassing.*
+import io.novafoundation.nova.feature_wallet_api.data.network.blockhain.types.Balance
+
+interface DelayedThresholdPassing {
+
+    class DelayedPassing(val delayFraction: Perbill, val passingInFuture: Boolean)
+
+    fun supportPassingInFuture(tally: Tally, totalIssuance: Balance, passedSinceDecidingFraction: Perbill): DelayedPassing
+
+    fun ayePassingInFuture(tally: Tally, totalIssuance: Balance, passedSinceDecidingFraction: Perbill): DelayedPassing
+}
```

Comment:
Tbh I dont think we need a separate interface for that, it could be a part of Threshold. Regarding the naming, i think we should use "projected" instead of "delayed"